package.json

```diff
@@ -55,14 +55,14 @@
     "nock": "9.2.3",
     "node-sass": "4.8.3",
     "nunjucks": "3.1.2",
-    "nyc": "11.6.0",
+    "nyc": "^11.7.3",
     "postcss-loader": "2.1.3",
     "postcss-url": "7.3.1",
     "regexp-replace-loader": "1.0.1",
     "sass-lint": "1.12.1",
     "sass-loader": "6.0.7",
     "schema-utils": "0.4.5",
-    "snyk": "1.70.3",
+    "snyk": "^1.80.1",
```

Comment:
one of the vulnerabilities was in a vulnerability checker? classic.

Reply:
Haha I know!